grid/sitemaps.py

```diff
@@ -0,0 +1,13 @@
+from django.contrib.sitemaps import Sitemap
+from .models import Grid
+
+
+class GridSitemap(Sitemap):
+    changefreq = "daily"
+    priority = 1.0
```

Comment:
```suggestion
    priority = 1.0
    protocol = "https"
```

Reply:
Since we’re on Django 5 do we need to be explicit with the protocol? Per the [docs](https://docs.djangoproject.com/en/5.0/ref/contrib/sitemaps/#django.contrib.sitemaps.Sitemap.protocol)

> Changed in Django 5.0:
> In older versions, the default protocol for sitemaps built outside the context of a request was 'http'.

Reply:
@jefftriplett I intensionally left it as is. Because:

> If protocol is None, the URLs in the sitemap will use the protocol with which the sitemap was requested.

This will make the views generate proper URLs both in production and in development. (Assuming we don't allow http on production)

Ref: https://github.com/django/django/blob/8c257cecffc314179a5375ab8a16ca0ff3b6fb16/django/contrib/sitemaps/__init__.py#L13

Reply:
Perfect. After ~18 years of it not working, I'm thrilled to learn that it finally "just works"